.ort.yml

```diff
@@ -10,3 +10,7 @@ excludes:
   - pattern: "test"
     reason: "TEST_DEPENDENCY_OF"
     comment: "Packages for testing only."
+license_choices:
+  repository_license_choices:
```

Comment:
Could you elaborate what does it do?

I read [the doc](https://github.com/oss-review-toolkit/ort/blob/f9f938b263bfe4ae3c8e8c45272bfdc65259035b/website/docs/configuration/ort-yml.md#license-choice-for-the-project), but I'm still not quite following.

Reply:
When scanning this project with OSS Review Toolkit, this file is picked up as configuration for this scan.
The added entry says: "Whenever you find a dependency with license `FTL OR GPL-2.0-or-later`m please choose `FTL` as license". This way, the policy engine checks only FTL against the policy rules and ignores the GPL, because of the license choice made.